petab/petab_schema.yaml

```diff
@@ -0,0 +1,46 @@
+# For syntax see: https://json-schema.org/understanding-json-schema/index.html
+#$schema: "https://json-schema.org/draft/2019-09/meta/core"
+$schema: "http://json-schema.org/draft-06/schema"
+description: PEtab parameter estimation problem config file schema
+
+properties:
+
+  petab_version:
+    type: string
+    description: Version of the PEtab format (e.g. 0.0.0a16)
+
+  parameter_file:
+    type: string
+    description: |
+      PEtab parameter file name containing parameters of all models listed in
+      `problems`. Absolute or relative path.
```

Comment:
I would directly allow general URLs here (including access mechanisms, e.g. http or file)

Reply:
Sounds useful. Will adapt the description. Implementation to be done at some later point. For the tables there shouldn't be any changes required except for file existance checks. For the SBML files, we'd have to implement it by ourselves.

Reply:
background: this will in particular allow using distributed and online sources, as with the jws repo.